### PR TITLE
convert CreateVOClient

### PR DESCRIPTION
**Pull request by edquist**
https://twiki.opensciencegrid.org/bin/view/SoftwareTeam/CreateVOClient

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

### CODE REVIEW

**Comment by edquist** [comment]
Note that this document is actually out of date.  I'd like to update it, but I think in a separate PR.

Also, the `#Automated_GUMS_Conversion` link does not seem to work properly - open to suggestions here.

**Comment by brianhlin** [comment]
Agreed, major doc updates should go in a separate PR.

File: docs/software/create-vo-client.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+-   Move the files into a subdirectory to include in the tarball:
+
+<!-- -->
+
+          VERSION=44  # set appropriately
+          mkdir vo-client-$VERSION
+          mv vomses gums.config.template edg-mkgridmap.conf vomsdir vo-client-$VERSION/
+          tar -czf vo-client-$VERSION-osg.tar.gz vo-client-$VERSION/
+
+Upload the tarball into the [upstream source cache](rpm-development-guide#Upstream_Source_Cache), in the<br> `vo-client/VERSION/` directory.
```
Comment (brianhlin) [inline docs/software/create-vo-client.md]: The subsection link is broken. There's also an extra `<br>`

File: docs/software/create-vo-client.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          # chmod +x convert-osg-gums-template-for-vo-client.sh
+          # ./convert-osg-gums-template-for-vo-client.sh gums.template
+
+-   It takes a little while to install and set up gums and related packages, but if it succeeds, you should see a message that says "User group has been saved.", and a file `gums.config.template` should be written in the current directory.
+-   The newly converted `gums.config.template` should be compared to the old version of that file (from the previous vo-client package) to ensure that the only the differences are the changes for this release. (I have had to manually strip the extra test account stuff.) The 'meld' program is a nice graphical diff tool that I use for comparing them.
+
+RPM spec file maintenance
+-------------------------
+
+The OSG RPM spec file is [maintained in Subversion](rpm-development-guide#Revision_Control_System).
```
Comment (brianhlin) [inline docs/software/create-vo-client.md]: Broken link

File: docs/software/create-vo-client.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+The VO Client package is located in<br> `native/redhat/trunk/vo-client`
```
Comment (brianhlin) [inline docs/software/create-vo-client.md]: Unnecessary line break

File: docs/software/create-vo-client.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+There are two files that need to be maintained:
+
+-   `osg/vo-client.spec` - This is the RPM spec file proper. One needs to update the version (and/or the release number) every time a new RPM is created.
+-   `upstream/release_tarball.source` - This file contains the relative path of the tarball within the [upstream source cache](rpm-development-guide#Upstream_Source_Cache). Since the tarball file name will change with every new RPM version, this file has to be changed accordingly.
```
Comment (brianhlin) [inline docs/software/create-vo-client.md]: Broken subsection link

**Comment by brianhlin** [comment]
@edquist please replace the twiki doc with the migration header

**Pull request merged**